test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('./lib/core');
+
+if (impress.isMaster) {
+  impress.on('tested', errors => {
```

Comment:
As I mentioned in the comment to `bailout` I suspect that this might not be called in case of unhandled exception.
Though needs more testing/info.